### DISCUSSION TITLE
Coloured and/or named wires

### DISCUSSION THREAD

**Discussion by GeoffreyParrier** [UkrainianBanderasCat/Digital-Logic-Sim2#65] [Ideas]
It can be a mess when there is a lot of wires, here some ideas to change that :
- Add the possibility to chose a color for a wire
- Add the possibility to name wires
- Add the possibility to hover a wire and show a tooltip of where it's from and where it goes
- Add the possibility to select a wire and change the colours of it, to know it's full path

**Comment by AlphaCodingPilot** [comment]
I dont know why you would want to name wires. Wires getting a random color assigned could be an option in the settings. Showing a tool tip for wires is a good idea.

**Reply by GeoffreyParrier** [reply]
Name is just a way to let people write something as a descriptive name.

For example : 
Multiple cables that do an equivalent thing in multiple places can have the same name.
